### DISCUSSION TITLE
Is there a way to export or embed graphs from https://cosmograph.app/run/

### DISCUSSION THREAD

**Discussion by bingbong-sempai** [cosmograph-org/cosmograph-issues#4] [Q&A] [answered]
Maybe as an html file?

**Comment by rokotyan** [comment]
@bingbong-sempai We've an undocumented graph sharing feature, here's an example URL: 

https://cosmograph.app/run/?nodeColor=color-size&nodeSize=size-size&data=https://gist.githubusercontent.com/banteg/14fa2d867958f7855a4c66b244682088/raw/02d3a4ac804226c2df9416a3b002e6327063a739/data.csv&meta=https://gist.githubusercontent.com/banteg/14fa2d867958f7855a4c66b244682088/raw/02d3a4ac804226c2df9416a3b002e6327063a739/metadata.csv

In theory, it should allow yo to embed a graph as an iframe, but we've never tried to do it ourselves.

**Reply by bingbong-sempai** [reply]
ahh thanks for this!

**Reply by bingbong-sempai** [reply]
By the way is there a list of parameters that can be set using the URL query string?  
Like would it be possible to set show labels / show links / node scale or set link width to "sum by weight" via query?

**Reply by rokotyan** [reply, marked as answer]
@bingbong-sempai Sure, here's a screenshot from our future documentation. As for showing/hiding labels and links, that's currently not supported, but we'll add that support in the future.

![QueryString_API___Cosmograph](https://github.com/cosmograph-org/cosmos/assets/755708/ad968594-8d65-472d-8ed1-cfb52ee8846e)

**Reply by bingbong-sempai** [reply]
Wow this is super useful, thank you so much!

**Comment by mr-tron** [comment]
is same option exists for nodeLabel? I tried nodeLabel=my_column_name_from_meta_file but it doesn't work

**Reply by rokotyan** [reply]
@mr-tron The feature doesn't exist at this moment, but once we add it we'll let you know.

**Reply by kolmakova** [reply]
Hi @mr-tron! We've added an option for `nodeLabel`. Here's a doc on how to use it: https://cosmograph.app/docs/cosmograph/Sharing%20Graphs#node-appearance
